crates/configuration/src/relaychain.rs

```diff
@@ -12,20 +12,20 @@ use crate::shared::{
 };
 
 /// A relay chain configuration, composed of nodes and fine-grained configuration options.
-#[derive(Debug, Clone, PartialEq, Serialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct RelaychainConfig {
     chain: Chain,
     default_command: Option<Command>,
     default_image: Option<Image>,
     default_resources: Option<Resources>,
     default_db_snapshot: Option<AssetLocation>,
     #[serde(skip_serializing_if = "std::vec::Vec::is_empty")]
-    default_args: Vec<Arg>,
+    pub(crate) default_args: Vec<Arg>,
     chain_spec_path: Option<AssetLocation>,
     random_nominators_count: Option<u32>,
     max_nominations: Option<u8>,
     #[serde(skip_serializing_if = "std::vec::Vec::is_empty")]
-    nodes: Vec<NodeConfig>,
+    pub(crate) nodes: Vec<NodeConfig>,
```

Comment:
Maybe we can replace this with a method like `set_nodes`

Reply:
```suggestion
    nodes: Vec<NodeConfig>,
```
we can make it private again here since you added the `set_nodes` fn